.github/workflows/create-docusaurus-pr.yml

```
@@ -37,9 +37,26 @@ jobs:
             echo "Source directory does not exist!"
             exit 1
           fi
+          # Remove old API docs but preserve other folders
           rm -rf docs-repo/docs/API/
           mkdir -p docs-repo/docs/API
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)